plenum/test/node_request/test_split_non_3pc_messages_on_batches.py

```diff
@@ -32,16 +21,18 @@ def test_msg_max_length_check_node_to_node(looper,
 
     patch_msg_len_validators(max_len_limit, txnPoolNodeSet)
 
-    client2, wallet2 = clientAndWallet2
+    reqs1 = sdk_send_random_requests(looper, sdk_pool_handle, sdk_wallet_client, N)
+    reqs2 = sdk_send_random_requests(looper, sdk_pool_handle, sdk_wallet_client2, N)
 
-    reqs1 = sendRandomRequests(wallet1, client1, N)
-    reqs2 = sendRandomRequests(wallet2, client2, N)
+    total_timeout = sdk_eval_timeout(N, len(txnPoolNodeSet))
 
-    check_reqacks(client1, looper, reqs1, txnPoolNodeSet)
-    check_reqacks(client2, looper, reqs2, txnPoolNodeSet)
+    sdk_replies = sdk_get_replies(looper, reqs1, timeout=total_timeout)
+    for req_res in sdk_replies:
+        sdk_check_reply(req_res)
 
-    waitForSufficientRepliesForRequests(looper, client1, requests=reqs1)
-    waitForSufficientRepliesForRequests(looper, client2, requests=reqs2)
+    sdk_replies = sdk_get_replies(looper, reqs2, timeout=total_timeout)
+    for req_res in sdk_replies:
+        sdk_check_reply(req_res)
```

Comment:
use special func to get and check replies instead of sdk_get_replies and for loop

Reply:
Done